bin/computecanada/partition-stats

```diff
@@ -1,4 +1,4 @@
-#!/cvmfs/soft.computecanada.ca/nix/var/nix/profiles/ruby-2.3.1/bin/ruby
+#!/bin/env ruby
```

Comment:
Is it safe to use an undetermined version of ruby ? Probably use `/cvmfs/soft.computecanada.ca/gentoo/2023/x86-64-v3/usr/bin/ruby` ?

Reply:
We only have `ruby31` in the env

Reply:
the ruby for the environment can change when a user loads a module or installs their own ruby

Reply:
Ah, unless a user loads ruby/3.2 and 3.2 breaks something...